lib/northbound.c

```diff
@@ -538,10 +543,16 @@ static inline void nb_config_diff_dnode_log(const char *context,
 }
 #endif
 
-/* Calculate the delta between two different configurations. */
-static void nb_config_diff(const struct nb_config *config1,
-			   const struct nb_config *config2,
-			   struct nb_config_cbs *changes)
+/*
+ * Calculate the delta between two different configurations.
+ *
+ * NOTE: 'config1' is the reference DB, while 'config2' is
+ * the DB being compared against 'config1'. Typically 'config1'
+ * should be the Running DB and 'config2' is the Candidate DB.
+ */
+void nb_config_diff(const struct nb_config *config1,
```

Comment:
given the comment above, it might make more sense to rename these to something other than `config1` and `config2` ... maybe `config_ref` and `config_compare` or something similar?

Reply:
Sure will rename these as per your suggestion. @ranjanyash54 please take care of addressing this comment.